openedx/features/course_bookmarks/views/course_bookmarks.py

```diff
@@ -65,6 +65,8 @@ class CourseBookmarksFragmentView(EdxFragmentView):
     """
     Fragment view that shows a user's bookmarks for a course.
     """
+    _uses_pattern_library = False
```

Comment:
Is this still needed?

Reply:
@abutterworth yes, it is for remove pattern library of the fragment view that comes from the EdxFragmentView.

Reply:
Yes, it is still needed, we will remove this line once the pattern_library is completely removed from the platform